CONTRIBUTING.md

```diff
@@ -103,13 +103,24 @@ The website's source code is in the `root` folder which contains content pages a
 ### Running the development environment
 
 #### Setting up Local HTTP/S environments
-  1. Navigate to solo/deploy/compose and run configure-local-environment.sh.
+  1. Navigate to solo/deploy/compose and run configure-local-environment.sh
       - `sh configure-local-environment.sh`
   2. Tell Chrome to allow self-signed certificates on localhost
       - Navigate to `chrome://flags/#allow-insecure-localhost` and enable that setting
-  3. Follow the directions given to you in the configure-local-environment.sh for building solo locally 
+  3. Follow the directions given to you in the configure-local-environment.sh for building solo locally
   after configuration.
   
+#### Setting up Local Proprietary Compression 
+  1. Navigate to solo/compression-service/EXML-converter-service and run pull-proprietary-code.sh (requires AWS authentication and region to be set)
+      - `sh pull-proprietary-code.sh.sh`
+  2. move up one directory
+      - ` cd ..`
+  3. Build and run the container and set the hostname of the container license ex:
```

Comment:
I would write out 'for example' vice 'ex'

Reply:
Agree, fixed.